neural_modelling/src/neuron/synapses.c

```diff
@@ -328,7 +349,25 @@ void synapses_flush_ring_buffers(timer_t time) {
 }
 
 bool synapses_process_synaptic_row(
-        uint32_t time, synaptic_row_t row, bool *write_back) {
+        uint32_t time, uint32_t spike_colour, uint32_t colour_mask,
+		synaptic_row_t row, bool *write_back) {
+
+    // Work out how much delay take off or add on to the actual delay because
+    // of a delayed spike arrival time, or delayed change of time step in the
+    // current core.  Spikes can be as late as the bits in colour_mask dictates.
+	/// Masked difference is used to calculate this, which will always be
```

Comment:
Looks like a stray / at the start of this line

Reply:
Fixed